types/ember-cli-fastboot/test/ember-cli-fastboot-tests.ts

```diff
@@ -0,0 +1,22 @@
+import { assertType } from './assert';
+import { FastBoot as FastBootService, FastbootRequest, Shoebox } from 'ember-cli-fastboot/services/fastboot';
+
+/** type assertions for global FastBoot object */
+assertType<FastBootService | undefined>(FastBoot);
```

Comment:
Per the comment above, this'll need to be different. I suggest we put the definition in a `-private` module and import it here under a different name, and then check that the global works. I *believe* we'll also then need to `import "ember-cli-fastboot"` to get it to resolve.

Reply:
updated! didn't need to do the import